rpm/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:24
+FROM fedora:27
```

Comment:
Given this image only exists to build .rpms, can we just use CentOS and churn less?

Reply:
CentOS should be fine yeah.